tests/test_load_legal_docs.py

```diff
@@ -0,0 +1,14 @@
+from mock import patch
+from data.load_legal_docs import *
+
+class ElasticSearchMock:
+    def search():
+        pass
+
+def get_es():
+    return ElasticSearchMock()
+
+class StatuteTest(unittest.TestCase):
+    @patch('webservices.utils.get_elasticsearch_connection', side_effect=get_es)
```

Comment:
Just out of curiosity, if you're using a `side_effect` that just returns a value, why not just use `return_value`?